security/pkg/stsservice/tokenmanager/tokenmanager.go

```diff
@@ -80,22 +80,24 @@ func GetGCPProjectInfo() GCPProjectInfo {
 
 // CreateTokenManager creates a token manager with specified type and returns
 // that token manager
-func CreateTokenManager(tokenManagerType string, config Config) security.TokenManager {
+func CreateTokenManager(tokenManagerType string, config Config) (security.TokenManager, error) {
 	tm := &TokenManager{
 		plugin: nil,
 	}
+	var err error
```

Comment:
```suggestion
```

Reply:
This error variable is needed because "if p, err := google.CreateTokenManagerPlugin(...)" may return an error.

Reply:
I have refactored the code to remove this variable.